resources/scripts/make-wallets.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Function to print a message in a box
+print_in_box() {
+    local message="$1"
+    local border_length=$((${#message} + 4))
+    local border=$(printf '%*s' "$border_length" | tr ' ' '#')
+
+    echo "$border"
+    echo "# $message #"
+    echo "$border"
+}
+
+# Check for correct number of arguments
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <number_of_wallets> <amount_per_wallet>"
+    exit 1
+fi
+
+# Get the input arguments
+NUM_WALLETS=$1
+AMOUNT_PER_WALLET=$2
+
+# Define directories relative to the current working directory
+APP_DIR=$(pwd)
+CLIENT_DIR="${APP_DIR}/client"
+NEW_CLIENT_DIR_TEMPLATE="${APP_DIR}/client_"
+
+# Initialize an array to store the wallet details
+declare -a WALLETS
+
+# Loop to create and fund wallets
```

Comment:
I think the whole flow can be optimized to:
1, backup the wallet with balance
2, repeated create 5 wallets, collecting their address
3, restore the wallet with balance, transfer fund to new wallets one by one

Not a big issue, as anyway scrit do the job